packages/google_maps_flutter/lib/src/polyline.dart

```diff
@@ -0,0 +1,216 @@
+part of google_maps_flutter;
+
+/// Uniquely identifies a [Polyline] among [GoogleMap] polylines.
+///
+/// This does not have to be globally unique, only unique among the list.
+@immutable
+class PolylineId {
+  PolylineId(this.value) : assert(value != null);
+
+  /// value of the [PolylineId].
+  final String value;
+
+  @override
+  bool operator ==(Object other) {
+    if (identical(this, other)) return true;
+    if (other.runtimeType != runtimeType) return false;
+    final PolylineId typedOther = other;
+    return value == typedOther.value;
+  }
+
+  @override
+  int get hashCode => value.hashCode;
+
+  @override
+  String toString() {
+    return 'PolylineId{value: $value}';
+  }
+}
+
+/// Configuration options for [Polyline] instances.
```

Comment:
I am guessing that this comment was before `PolylineOptions` were made part of `Polyline`. Please fix this to reflect the current nature of the widget.